README.md

```diff
@@ -73,13 +75,13 @@ The ScheduleEntry object requires the following name/value pairs:
 
 |name|required|type|unit|description|
 |----|--------------|-------|-------|-----------|
-|`name`|true|string|N/A|The unique identification string assigned to the schedule entry.|
-|`start`|false|integer|seconds|Absolute start time of the schedule in [Unix time](https://en.wikipedia.org/wiki/Unix_time).|
-|`relative_stop`|false|integer|seconds|Stop time of the schedule relative to `start` time.|
-|`stop`|false|integer|seconds|Absolute stop time of the schedule in [Unix time](https://en.wikipedia.org/wiki/Unix_time).|
-|`interval`|false|integer|seconds|Interval time between instances of the `action` being performed.|
-|`priority`|false|integer|N/A|Priority of the schedule, similar to applying [nice](https://en.wikipedia.org/wiki/Nice_(Unix)). Lower numbers are higher priority.|
-|`Action`|true|string|N/A|Name of action to be performed.|
+|`name`|true|string|N/A|The identification string assigned to the schedule entry. MUST be unique on the sensor.|
+|`start`|false|integer|seconds|Requested time to schedule the first task in [Unix time](https://en.wikipedia.org/wiki/Unix_time). Default if unspecified is to start as soon as received.|
+|`relative_stop`|false|integer|seconds|`stop` should be interpreted as seconds after `start`. Default is false.|
```

Comment:
`relative stop` instead of `stop` might be clearer.

Reply:
I do mean `stop` here. I'm trying to say, succinctly, that if `relative_stop` is true, then the value in `stop` should be interpreted as seconds after `start`, instead of an absolute time.

Reply:
Maybe we can just say that if being succinct is at all confusing.

Reply:
Ok, sorry, I see what you're saying now. Better?:

`stop` name/value pair will be interpreted as seconds after `start`

Reply:
Changing relative_stop to a boolean makes sense to me

Reply:
I'm not sure that makes it clearer? What does it mean for a "name/value pair to be interpreted as second". Also, why wouldn't you also use that after `start`?

If my description above was clearer, maybe we could just go with that?

Reply:
Nevermind, I just noticed @MichaelCotton 's comment that `relative_stop` is listed as integer in the spec. That's not correct, it is boolean. I can understand the confusion now.